tests/interface/TWCoinTypeConfigTests.cpp

```diff
@@ -231,7 +231,7 @@ TEST(TWCoinTypeConfiguration, TWCoinTypeConfigurationGetTransactionURL) {
     assertStringsEqual(tron, "https://tronscan.org/#/transaction/123");
 
     auto vet = WRAPS(TWCoinTypeConfigurationGetTransactionURL(TWCoinTypeVeChain, txId));
-    assertStringsEqual(vet, "https://explore.veforge.com/transactions/123");
+    assertStringsEqual(vet, "https://insight.vecha.in/#/txs/123");
```

Comment:
```suggestion
    assertStringsEqual(vet, "https://insight.vecha.in/#/txs/0x34cf419333786e22d78eae312d03670938eabf2819dbc23b826c15b60221d030");
```

Reply:
we reuse `123` as a dummy tx id, we can refactor this part later

Reply:
https://github.com/trustwallet/wallet-core/issues/694